ravenframework/MessageHandler.py

```diff
@@ -19,19 +19,11 @@
 import sys
 import time
 import bisect
-import builtins
 
 from .utils import utils
 
 _starttime = time.time()
 
-#custom exceptions
-class NoMoreSamplesNeeded(GeneratorExit):
```

Comment:
watching for where this got moved to ...

Reply:
This already existed in `utils`. Every reference I found to `NoMoreSamplesNeeded` used the one from `utils`.